src/read.c

```diff
@@ -2197,7 +2195,7 @@ avifResult avifDecoderSetIOFile(avifDecoder * decoder, const char * filename)
 {
     avifIO * io = avifIOCreateFileReader(filename);
     if (!io) {
-        return AVIF_RESULT_NO_IO;
+        return AVIF_RESULT_IO_ERROR;
```

Comment:
I changed the error code because "IO error" seems like the closest approximation to "file not found".